schema/json.go

```diff
@@ -63,6 +63,19 @@ func (dst *JSON) Set(src interface{}) error {
 		} else {
 			*dst = JSON{Bytes: value, Status: Present}
 		}
+	// In the case of map[string]string (fields like 'Tags'), an empty map is easier for users to work with than
+	// postgres "null", or JSON "null".
+	case map[string]string:
+		if len(value) == 0 {
+			*dst = JSON{Bytes: []byte("{}"), Status: Present}
+			break
+		}
+		buf, err := json.Marshal(value)
```

Comment:
You could move this `case` just before the `default` and use `fallthrough` keyword but would make it harder to read and reason/refactor later on...

Reply:
Agree :)